services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/assets/index.module.scss

```diff
@@ -59,6 +59,7 @@
   display: flex;
   width: 100%;
   padding-left: spacing.$spacing-05;
+  border-bottom: 1px solid theme.$border-subtle;
```

Comment:
is this one necessary I see a weird mismatch now in the gray part (see to the right, where it ends)
![image](https://user-images.githubusercontent.com/40550942/184916197-33797afe-c10f-4d1b-8d18-1da813368688.png)

Reply:
Good catch, removed!